sysID/fit_flywheel_philip.py

```diff
@@ -0,0 +1,180 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from models import cartpole_flywheel as model
+
+# Load the data
+q = np.load('/Users/philipqueen/Documents/Humon Research Lab/FreeMocap_Data/4stepsequence_session2_10_5_22/pendulum_flywheel_state_q.npy')
+qDot = np.load('/Users/philipqueen/Documents/Humon Research Lab/FreeMocap_Data/4stepsequence_session2_10_5_22/pendulum_flywheel_state_qdot.npy')
+qDotDot = np.load('/Users/philipqueen/Documents/Humon Research Lab/FreeMocap_Data/4stepsequence_session2_10_5_22/pendulum_flywheel_state_qddot.npy')
+
+params = model.default_params()
+
+# convert to meters
+q[:, 0] = q[:, 0]/1000.
+
+qDot[:, 0] = qDot[:, 0]/1000.
+
+qDotDot[:, 0] = qDotDot[:, 0]/1000.
+
+# convert angle to radians
+q[:, 1] = q[:, 1]*np.pi/180.
+qDot[:, 1] = qDot[:, 1]*np.pi/180.
+qDotDot[:, 1] = qDotDot[:, 1]*np.pi/180.
+
+# convert to meters squared
+q[:, 2] = q[:, 2]/1000000.
+qDot[:, 2] = qDot[:, 2]/1000000.
+qDotDot[:, 2] = qDotDot[:, 2]/1000000.
+
+# multiply by mass to get kg/m^2
+q[:, 2] = q[:, 2] * params["mass_pendulum"]
```

Comment:
similar to above, but depends what your reasoning is here.
These units seem like you're calculating pressure or something...